src/palace/manager/api/axis/parser.py

```diff
@@ -52,155 +18,42 @@
-
-    def process_one(
-        self, tag: _Element, namespaces: dict[str, str] | None
-    ) -> tuple[int, str] | None:
-        status_code = self.int_of_subtag(tag, "axis:code", namespaces)
-        message = self.text_of_subtag(tag, "axis:statusMessage", namespaces)
-        return status_code, message
-
-    def process_first(
-        self,
-        xml: str | bytes | _ElementTree | None,
-    ) -> tuple[int, str] | None:
-        if not xml:
-            return None
-
-        # Since this is being used to parse error codes, we want to generally be
-        # very forgiving of errors in the XML, and return None if we can't parse it.
-        try:
-            return super().process_first(xml)
-        except (etree.XMLSyntaxError, AssertionError, ValueError):
-            return None
-
-
-class BibliographicParser(
-    Axis360Parser[tuple[BibliographicData, CirculationData]], LoggerMixin
-):
-    DELIVERY_DATA_FOR_AXIS_FORMAT = {
-        Axis360APIConstants.BLIO: None,  # Legacy format, handled the same way as AxisNow
-        "Acoustik": (None, DeliveryMechanism.FINDAWAY_DRM),  # Audiobooks
-        Axis360APIConstants.AXISNOW: None,  # Handled specially, for ebooks only.
-        "ePub": (Representation.EPUB_MEDIA_TYPE, DeliveryMechanism.ADOBE_DRM),
-        "PDF": (Representation.PDF_MEDIA_TYPE, DeliveryMechanism.ADOBE_DRM),
-    }
-
-    @classmethod
-    def parse_list(cls, l: str) -> list[str]:
-        """Turn strings like this into lists:
-
-        FICTION / Thrillers; FICTION / Suspense; FICTION / General
-        Ursu, Anne ; Fortune, Eric (ILT)
-        """
-        return [x.strip() for x in l.split(";")]
-
-    @property
-    def xpath_expression(self) -> str:
-        return "//axis:title"
-
-    def extract_availability(
-        self,
-        circulation_data: CirculationData | None,
-        element: _Element,
-        ns: dict[str, str] | None,
-    ) -> CirculationData:
-        identifier = self.text_of_subtag(element, "axis:titleId", ns)
-        primary_identifier = IdentifierData(
-            type=Identifier.AXIS_360_ID, identifier=identifier
-        )
-        if not circulation_data:
-            circulation_data = CirculationData(
-                data_source_name=DataSource.AXIS_360,
-                primary_identifier_data=primary_identifier,
-            )
 
-        availability = self._xpath1(element, "axis:availability", ns)
-        total_copies = self.int_of_subtag(availability, "axis:totalCopies", ns)
-        available_copies = self.int_of_subtag(availability, "axis:availableCopies", ns)
-        size_of_hold_queue = self.int_of_subtag(availability, "axis:holdsQueueSize", ns)
 
-        circulation_data.licenses_owned = total_copies
-        circulation_data.licenses_available = available_copies
-        circulation_data.licenses_reserved = 0
-        circulation_data.patrons_in_hold_queue = size_of_hold_queue
-
-        return circulation_data
+class BibliographicParser(LoggerMixin):
```

Comment:
This takes the parsed Availability response and turns it into `BibliographicData` and `CirculationData` tuples used by the CM.